dbms/src/Storages/DeltaMerge/SkippableBlockInputStream.h

```diff
@@ -219,18 +223,15 @@ class ConcatSkippableBlockInputStream : public SkippableBlockInputStream
         if (likely(scan_context != nullptr))
         {
             scan_context->total_user_read_bytes += bytes;
-
-            if (scan_context->enable_resource_control)
-                LocalAdmissionController::global_instance->consumeResource(
-                    scan_context->resource_group_name,
-                    bytesToRU(bytes),
-                    0);
+            assert(lac_bytes_collector);
+            lac_bytes_collector->collect(bytes);
         }
     }
     BlockInputStreams::iterator current_stream;
     std::vector<size_t> rows;
     size_t precede_stream_rows;
     const ScanContextPtr scan_context;
+    LACBytesCollectorPtr lac_bytes_collector;
```

Comment:
`ConcatSkippableBlockInputStream` and `ColumnFileSetReader` will be destroyed immediately after execution instead of waiting for mpptask to be destroyed, right? @JinheLin